### ISSUE TITLE
Issues with creating shortcuts from Castbox

### ISSUE THREAD

**Issue by HunterArmstrong** [BranchMetrics/sesame_issues#179]
Currently there are no shortcuts from Castbox that I can add to Sesame. Idk how but. I feel like making shortcuts should be easier than that

**Comment by philwall2** [comment]
We have a few automatic ways to make shortcuts built in. If Castbox isn't showing them it means they haven't set up their data to be available for shortcuts.

That leaves 2 options:

1. If they have an API we could use that
2. We look into their code and see if we could hack any (only works sometimes)

How much do you use Castbox? Is it worth all that effort? If so, what shortcuts would you want?

**Comment by HunterArmstrong** [comment]
For me, I use castbox every day. When you ask me how important castbox is. I would say it's a pretty good app. With over 10 million downloads, I would say it is a thing worth making shortcuts for. For me, I would mainly have my podcast subscriptions as my shortcuts, typically. 

Can I also ask, how often do you update Sesame?

**Comment by philwall2** [comment]
Aight, we'll look at Castbox and other podcast apps.

We used to update all the time. Over the past year we've taken on some big projects that slowed down releases (UI options, building the data library for launchers, contact actions).

Right now we're working on another big feature (voice related). Once that comes out we're hoping to make a bunch of quick releases / improvements.

**Comment by HunterArmstrong** [comment]
Yeah. Voice assistant seems like a huge thing. I hope it all comes out well and the others follow. 😁

**Comment by HunterArmstrong** [comment]
> Aight, we'll look at Castbox and other podcast apps.

Also, [here's the link](https://play.google.com/store/apps/details?id=fm.castbox.audiobook.radio.podcast) to where I got castbox
